ios/Peepz/Peepz/PeepzApp.swift

```diff
@@ -0,0 +1,29 @@
+import SwiftUI
+import GoogleSignIn
+import Firebase
+
+class AppDelegate: NSObject, UIApplicationDelegate {
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey : Any]? = nil) -> Bool {
+
+        FirebaseApp.configure()
+
+        return true
+    }
+
+    func application(_ application: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey : Any]) -> Bool {
+        return GIDSignIn.sharedInstance().handle(url)
+    }
+}
```

Comment:
Im starting to think this could be moved to `PeepzModel()` initialization, need to look closer to app life cycle.Add ticket for this..